web/libs/frontend-test/src/helpers/LSF/ToolBar.ts

```diff
@@ -1,6 +1,16 @@
+import { FF_DEV_1170 } from "@humansignal/frontend-test/feature-flags";
+import { LabelStudio } from "@humansignal/frontend-test/helpers/LSF/LabelStudio";
+import { FF_DEV_3873 } from "../../../../editor/src/utils/feature-flags";
+
 export const ToolBar = {
   get root() {
-    return cy.get(".lsf-topbar");
+    return LabelStudio.getFeatureFlag(FF_DEV_3873).then((isFFDEV3873) => {
```

Comment:
can we safely switch to `.lsf-bottombar` and all tests will work? assuming that we will have `FF_DEV_3873` enabled all the time

Reply:
I'll address this [in my tests](https://github.com/HumanSignal/label-studio/pull/6019)